Sources/MediaViewController.swift

```diff
@@ -61,9 +64,21 @@ public class VLCMediaViewController: UICollectionViewController, UISearchResults
         setupCollectionView()
         setupSearchController()
         setupNavigationBar()
+        setupRendererDiscovererManager()
         _ = (MLMediaLibrary.sharedMediaLibrary() as! MLMediaLibrary).libraryDidAppear()
     }
 
+    public override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        let manager = VLCRendererDiscovererManager.sharedInstance
```

Comment:
this code doesn't belong for me here. I would expect the manager to live inside the services

Reply:
You're right, I wasn't aware of the `VLCService` class.